### PR TITLE
Adapt Docker-Compose files to finitequiz and improve visual appearance

### PR DESCRIPTION
**Pull request by LeviUniAcc**

### CODE REVIEW

**Review by marlau97 — APPROVED** [review]

File: src/App.vue [hunk 1]
```diff
@@ -1,8 +1,15 @@
 <template>
   <div>
     <nav class="navbar navbar-expand-lg navbar-light bg-dark">
-      <a id="FiniteGame">FINITE GAME</a>
-      <b-button id="ResetGame" v-on:click="relaodPage">Restart</b-button>
+      <a id="FiniteGame">Finite Quiz</a>
+      <nav class="ms-auto">
+        <b-button class="NavButtons" id="RestartButton" v-on:click="relaodPage">
```
Comment (delvh) [inline src/App.vue]: typo

Comment (LeviUniAcc) [reply src/App.vue]: done

File: src/App.vue [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
   <div>
     <nav class="navbar navbar-expand-lg navbar-light bg-dark">
-      <a id="FiniteGame">FINITE GAME</a>
-      <b-button id="ResetGame" v-on:click="relaodPage">Restart</b-button>
+      <a id="FiniteGame">Finite Quiz</a>
+      <nav class="ms-auto">
+        <b-button class="NavButtons" id="RestartButton" v-on:click="reloadPage">
+          Restart
+        </b-button>
+        <b-button class="NavButtons" id="CloseButton" v-on:click="closeGame">
```
Comment (delvh) [inline src/App.vue]: You have weird ideas about how classes and IDs should look like…
They should follow [kebab-case](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/etc/text-cases.html), and not PascalCase…
Same thing everywhere else

**Review by delvh — APPROVED** [review]

File: src/App.vue (hunk 2 above)
Comment (LeviUniAcc) [reply src/App.vue]: done

**Pull request closed without merging**